common/beerocks/bwl/common/ap_wlan_hal.h

```diff
@@ -113,7 +113,7 @@ class ap_wlan_hal : public virtual base_wlan_hal {
      * @param [in] mac The MAC address of the station.
      * @param [in] bssid The MAC address of the target AP.
      * @param [in] chan The channel of the target AP.
-     * @param [in] disassoc_timer Time in milliseconds before the AP should 
+     * @param [in] disassoc_timer Time in TU (time units, 1 TU = 1.024ms) before the AP should 
```

Comment:
To complete the transition, we could change the argument name (here and in all implementations) into `disassoc_timer_ms` as well. (nitpick)